fastlane.gemspec

```diff
@@ -45,7 +45,7 @@ Gem::Specification.new do |spec|
     "docs_url" => "https://docs.fastlane.tools"
   }
 
-  spec.required_ruby_version = '>= 2.0.0'
+  spec.required_ruby_version = '>= 2.4.0'
```

Comment:
IMO, we can't do this without a major version bump. This could potentially break a lot of setups where Ruby 2.4 isn't installed by default and a bunch of CIs. If/when we are going to do this, we should really add a warning for a while first saying that this is going to happen.

Reply:
I agree with this being a major version bump @joshdholtz. Although I’m not sure we need an advance warning, since nobody is forcing them to upgrade and the `update_fastlane` action will ignore major version bumps.